app/helpers/stories_helper.rb

```diff
@@ -1,2 +1,18 @@
 module StoriesHelper
+  def status_label(story)
+    "<span class='story-status-badge #{story.status}'>#{story.status}</span>".html_safe
```

Comment:
I'd like to see the status with upper case in the first letter as we saw it in the select field.

Reply:
I think I'm going to skip this one since I think labels are better in lowercase but, mainly, because when I started to change this, I noticed I'd have to edit quite a few specs and a bit of JS that was swapping css classes based on the text content of the label.

Maybe add this as a bug or improvement story for later on?